health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/SearchChecklist.js

```
@@ -39,12 +39,28 @@ const SearchChecklist = () => {
+  const reqCriteria = {
+    url: `/mdms-v2/v2/_search`,
+    body: {
+      MdmsCriteria: {
+        tenantId: tenantId,
+        schemaCode: "HCM-ADMIN-CONSOLE.rolesForChecklist"
+      }
+    }
+  };
+  const { isLoading1, data: dataBT, isFetching1 } = Digit.Hooks.useCustomAPIHook(reqCriteria);
+  useEffect(() => {
+    let data = dataBT?.mdms;
```

Biome

[error] This let declares a variable that is only assigned once.

'data' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)